cores/nRF5/wiring.c

```diff
@@ -34,6 +34,7 @@ void init( void )
   NRF_CLOCK->LFCLKSRC = (uint32_t)((CLOCK_LFCLKSRC_SRC_RC << CLOCK_LFCLKSRC_SRC_Pos) & CLOCK_LFCLKSRC_SRC_Msk);
   NRF_CLOCK->TASKS_LFCLKSTART = 1UL;
 
+  NRF_RTC1->INTENSET = RTC_INTENSET_OVRFLW_Msk;
   NRF_RTC1->PRESCALER = 0;
   NRF_RTC1->EVTENSET = offsetof(NRF_RTC_Type, EVENTS_OVRFLW);
   NRF_RTC1->INTENSET = offsetof(NRF_RTC_Type, EVENTS_OVRFLW);
```

Comment:
Things are really busted now, this whole block needs to be replaced with:

```c
  NRF_RTC1->PRESCALER = 0;
  NRF_RTC1->EVTENSET = RTC_INTENSET_OVRFLW_Msk;
  NRF_RTC1->INTENSET = RTC_EVTEN_OVRFLW_Msk;
  NRF_RTC1->TASKS_START = 1;
```

As ` NRF_RTC1->INTENSET` get overwritten with `offsetof(NRF_RTC_Type, EVENTS_OVRFLW);` which is completely wrong. After waiting over 512 seconds, the `millis()` value wrapped with the following sketch:

```arduino
void setup() {
  Serial.begin(9600);

  Serial.println("NRF_RTC1->INTENSET");
  Serial.println(NRF_RTC1->INTENSET, HEX);
}

void loop() {
  Serial.println(millis());
  delay(1000);
}
```

@austin1611 could you please make this change and test again? After that, we are good this merge this!

Thanks.